cms/celery.py

```diff
@@ -34,6 +34,5 @@ def alternate_env_tasks(self):
         return {
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache': 'lms',
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache_v2': 'lms',
-            'openedx.core.djangoapps.grades.tasks.compute_grades_for_course': 'lms',
```

Comment:
@sanfordstudent I think we can leave it here for now -- there's apparently a ticket on the horizon to move the grades app to 'openedx', so we can refactor this then.